noxfile.py

```diff
@@ -29,7 +29,7 @@
 
 DEFAULT_PYTHON_VERSION = "3.8"
 SYSTEM_TEST_PYTHON_VERSIONS = ["3.8"]
```

Comment:
Late to the party, but shouldn't we have bumped the systest version, too? :slightly_smiling_face: